### PR TITLE
JITMs for Plugins and Add New screens

### PR DESCRIPTION
**Pull request by eliorivero**
JITM for Plugins screen is only displayed after a plugin is activated.

JITM for Add New screen is only displayed for posts and pages.

### CODE REVIEW

File: class.jetpack-jitm.php [hunk 1]
```diff
@@ -98,6 +126,88 @@ function photon_msg() {
...
+		}
+	}
+
+	/**
+	 * Display message in editor prompting user to compose entry in WordPress.com.
+	 *
+	 * @since 3.8.2
+	 */
+	function editor_msg() {
+		$post_type = get_post_type();
```
Comment (dereksmart) [inline class.jetpack-jitm.php]: get_post_type is supposed to be used <a href="https://codex.wordpress.org/Function_Reference/get_post_type">within the loop</a>.  I'd suggest getting the post type with the global `$typenow` from within the admin.

Comment (eliorivero) [inline class.jetpack-jitm.php]: Changed.

**Comment by jeffgolenski** [comment]
Styles look good. Thanks @eliorivero !

**Comment by eliorivero** [comment]
Thanks for reviewing @jeffgolenski :+1:

**Pull request merged**